Python/RSA_Key_Generation/RSA.py

```diff
@@ -0,0 +1,148 @@
+
+
+
+if __name__ == '__main__':
+    while True:
+        command= input().strip().split(' ')
+
+        # if the user demands to generate the key pair
+        if command[0] == 'generate_keys':
+            try:
+                filePath_for_Keys= command[1]
+                kpg= keyPairGeneration(filePath_for_Keys)
+            except:
+                print ("Please verify the file path(s) given.")
+
+        # if the user demands to encrypt some confidential file
+        elif command[0] == 'encrypt':
+            try:
+                # extract the path of the public key file
+                filePath_of_PublicKey= command[1]
+                # extract the path of the file to be encrypted
+                file_to_encrypt= command[2]
+                # extract the filename user demands to give to will be formed encrypted file
+                encrypted_filename= command[3]
+
+                # creating object of utilities to access
+                util= Utilities()
+
+                # read the public key
+                publicKey= util.readFile(filePath_of_PublicKey)
+                # read the to be encrypted content
+                unencrypted_blob= util.readFile(file_to_encrypt)
+
+                rsa= RSA_algorithm()
+                # encrypt the content
+                encrypted_blob= rsa.encrypt(publicKey, unencrypted_blob)
+                # write the encrypted blob to get the encrypted contents of the file
+                util.writeFile(encrypted_filename, encrypted_blob)
+
+            except:
+                print ("Please verify the file path(s) given.")
+
+        # if the user demands to decrypt some encrypted file
+        elif command[0] == 'decrypt':
+            try:
+                # extract the path of the private key file
+                filePath_of_PrivateKey= command[1]
+                # extract the path of the file to be decrypted
+                file_to_decrypt= command[2]
+                # extract the filename user demands to give to will be formed decrypted file
+                decrypted_filename= command[3]
+
+                # creating object of utilities to access
+                util= Utilities()
+
+                # read the private key
+                privateKey= util.readFile(filePath_of_PrivateKey)
+                # read the to be decrypted content
+                encrypted_blob= util.readFile(file_to_decrypt)
+
+                rsa= RSA_algorithm()
+                # decrypt the content
+                decrypted_blob= rsa.decrypt(privateKey, encrypted_blob)
+                # write the decrypted blob to get the original contents of file
+                util.writeFile(decrypted_filename, decrypted_blob)
+
+            except:
+                print ("Please verify the file path(s) given.")
+
+        # user demads to exit from the program
+        elif command[0] == 'exit':
+            break
+
+        else:
+            print ("Invalid Command, please verify from the manual.")
+    
```

Comment:
Remove extra line. Add a new line at the EOF

Reply:
Can you please be more specific about which extra line are you talking about?
Also by adding new line at EOF, do you mean adding an extra blank line at the bottom of the file?